0001/README.md

```diff
@@ -0,0 +1,398 @@
+defined.
+
+Documents under active discussion should be in the `discussion` state. At this
+point a discussion is being had for the RFD in a Pull Request.
+
+Once (or if) discussion has converged and the Pull Request is ready to be
+merged, it should be updated to the published state before merge. Note that just
+because something is in the `published` state does not mean that it cannot be
+updated and corrected. See the Making changes to an RFD section for more
+information.
+
+The `prediscussion` state should be viewed as essentially a collaborative
+extension of an engineer's notebook, and the `discussion` state should be used
+when an idea is being actively discussed. These states shouldn't be used for
+ideas that have been committed to, organizationally or otherwise; by the time an
+idea represents the consensus or direction, it should be in the `published` state.
+
+Once an idea has been entirely implemented, it should be in the `committed` state.
+Comments on ideas in the `committed` state should generally be raised as issues --
+but if the comment represents a call for a significant divergence from or
+extension to committed functionality, a new RFD may be called for; as in all
+things, use your best judgment.
+
+Finally, if an idea is found to be non-viable (that is, deliberately never
+implemented) or if an RFD should be otherwise indicated that it should be
+ignored, it can be moved into the `abandoned` state.
+
+We will go over this in more detail. Let's walk through the life of a RFD.
+
+### RFD life-cycle
+
+There is a prototype script in this repository, scripts/new.sh, that will
+automate the process.
+
+```sh
+$ scripts/new.sh 0042 "My title here"
+```
+
+If you wish to create a new RFD by hand, or understand the process in greater detail, read on.
+
+NOTE: Never at anytime through the process do you push directly to the master
+branch. Once your pull request (PR) with your RFD in your branch is merged into
+master, then the RFD will appear in the master branch.
+
+### RESERVE A RFD NUMBER
+
+You will first need to reserve the number you wish to use for your RFC. This
+number should be the next available RFD number from looking at the current git
+branch -r output.
+
+### CREATE A BRANCH FOR YOUR RFD
+
+Now you will need to create a new git branch, named after the RFD number you
+wish to reserve. This number should have leading zeros if less than 4 digits.
+Before creating the branch, verify that it does not already exist:
+
+```sh
+$ git branch -rl *0042
+```
+
+If you see a branch there (but not a corresponding sub-directory in rfd in
+master), it is possible that the RFD is currently being created; stop and check
+with co-workers before proceeding! Once you have verified that the branch
+doesn't exist, create it locally and switch to it:
+
+```sh
+$ git checkout -b 0042
+```
+
+### CREATE A PLACEHOLDER RFD
+Now create a placeholder RFD. You can do so with the following commands:
```

Comment:
This is a nitpick, but the spacing around this header is not consistent.
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines

Reply:
this is solved, not sure why GitHub does not recognize it